arch/arm/src/sama5/Kconfig

```diff
@@ -3580,25 +3584,28 @@ if !SAMA5_ADC_ANARCH
 config SAMA5_ADC_GAIN
 	int "Analog gain"
 	default 1
-	depends on SAMA5_ADC_CHAN0
+	depends on ARCH_CHIP_SAMA5D3
 	range 0 3
 	---help---
 		Valid gain settings are {0, 1, 2, 3} which may be interpreted as
 		either {1, 1, 2, 4} if the channels are configured for single ended
 		mode or as {0.5, 1, 2, 2} if the channels are configured for
-		differential mode.
+		differential mode. Applies to all channels.
 
 config SAMA5_ADC_OFFSET
 	bool "Offset"
 	default n
+    depends on ARCH_CHIP_SAMA5D3
```

Comment:
TABs

Reply:
All lined up in VS Code. Have deleted and reinserted and its still lined up.